src/organizercore.cpp

```diff
@@ -862,9 +862,10 @@ QStringList OrganizerCore::findFiles(
     dir = dir->findSubDirectoryRecursive(ToWString(path));
   if (dir != nullptr) {
     std::vector<FileEntryPtr> files = dir->getFiles();
-    foreach (FileEntryPtr file, files) {
-      if (filter(ToQString(file->getFullPath()))) {
-        result.append(ToQString(file->getFullPath()));
+    for (FileEntryPtr &file: files) {
+      QString fullPath = QString::fromStdWString(file->getFullPath());
```

Comment:
Why not keep using ToQstring() ? Internally it uses fromStdWString but also has an overload for fromStdString depending on the passed argument. Of course skipping the second conversion is good.

Reply:
If it were for me, I would get rid of `ToQString` everywhere. :) 

I don't really see the point in all these methods from `uibase` that have equivalent in Qt. It's not like `->getFullPath` is going to change any time soon, and if it does, I might not want it to compile silently.

Reply:
I think the point was to not have to worry everywhere whether it's a wstring or normal string. We recently managed to increase performance a lot by changing these two functions. I doubt it will be the case but if there is ever a better alternative to QString::fromStdWString we could just change it in uibase and all the code would be updated. At least that is what happened with the previous implementation. I don't know how likely it is to happen again